joomla-projects/joomla-es6
# Remove inline Javascript for administrator/components/com_languages/tmpl/override/edit.php

Pull Request for Issue # https://github.com/joomla-projects/joomla-es6/issues/55.

### Summary of Changes
Remove inline Javascript for administrator/components/com_languages/tmpl/override/edit.php


### Testing Instructions
If you like to test you can open the override view in com_language and insert a search string and see that Joomla.overrider.refreshCache() is refreshed if expired.
